src/App.js

```diff
@@ -1,29 +1,14 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+// eslint-disable-next-line import/extensions
+import Calculator from './components/Calculator.js';
```

Comment:
- [ ] Kindly remove eslint disabling here also.

  For the first one use suggestion above. For the second one you need to define the import correctly to match the file name (small letter "c", not the capital letter "C"). You may get an extension error - just remove `.js`.

  ```
  import Calculator from './components/calculator';
  ```